env/.env.local

```diff
@@ -1,5 +1,5 @@
 REACT_APP_STAGE=LOCAL
 REACT_APP_VERSION=GITHUB_RUN_NUMBER
-REACT_APP_CHALLENGE_BASE_URL=https://pokeapi.co/api/v2
+REACT_APP_CHALLENGE_BASE_URL=https://astrostreakstest.co/api
```

Comment:
if this is running locally on the machine it may be localhost and not the domain name?

Reply:
Yep, once backend is running, we can switch this to localhost